src/vs/workbench/api/browser/mainThreadNotebookKernels.ts

```diff
@@ -90,6 +90,10 @@ abstract class MainThreadKernel implements INotebookKernel {
 			this.implementsExecutionOrder = data.supportsExecutionOrder;
 			event.hasExecutionOrder = true;
 		}
+		if (data.supportsInterrupt !== undefined) {
```

Comment:
is there a scenario for when a kernel goes from supporting Interrupt to not supporting Interrupt?

Reply:
👍 we should handle this, and also for other properties too `supportsExecutionOrder`. Will have follow up PRs for this.